src/test/e2e/tests/specs.test.ts

```diff
@@ -275,62 +228,64 @@ if (process.env.EXCLUDE_E2E === 'true') {
+      expect(await signInPage.getPageTitle()).toEqual('How do you want to sign in?');
+    });
+
+    it('should see 3 radio buttons', async () => {
+      expect(await signInPage.radioButtons).toBe(3);
+    });
 
-      it('should open sign-in page with \'How do you want to sign in\' title', async () => {
+    describe('sign in process and page routing', async () => {
+      it('should select \'Sign in with My HMCTS\' option and navigate to the login page HMCTS page', async () => {
         await signInPage.open('/sign-in');
-        expect(await signInPage.getPageTitle()).toEqual('How do you want to sign in?');
+        await signInPage.selectOption('SignInRadio1');
+        expect(await signInPage.clickContinueForRadio1()).toHaveHref(HMCTSAccountUrl);
       });
 
-      it('should see 3 radio buttons', async () => {
-        expect(await signInPage.radioButtons).toBe(3);
+      it('should select \'Sign in with Common Platform\' option and navigate to the login page Common Platform page', async () => {
+        await signInPage.open('/sign-in');
+        await signInPage.selectOption('SignInRadio2');
+        expect(await signInPage.clickContinueForRadio2()).toHaveHref(HMCTSAccountUrl);
       });
 
-      describe('sign in process and page routing', async () => {
-        it('should select \'Sign in with My HMCTS\' option and navigate to the login page HMCTS page', async () => {
-          await signInPage.open('/sign-in');
-          await signInPage.selectOption('SignInRadio1');
-          expect(await signInPage.clickContinueForRadio1()).toHaveHref(HMCTSAccountUrl);
-        });
-
-        it('should select \'Sign in with Common Platform\' option and navigate to the login page Common Platform page', async () => {
-          await signInPage.open('/sign-in');
-          await signInPage.selectOption('SignInRadio2');
-          expect(await signInPage.clickContinueForRadio2()).toHaveHref(HMCTSAccountUrl);
-        });
-
-        it('should select \'Sign in with my P&I details\' option, navigate to the login page, and sign in', async () => {
-          await signInPage.open('/sign-in');
-          await signInPage.selectOption('SignInRadio3');
-          await signInPage.clickContinueForRadio3();
-          console.log('B2C_USERNAME', config.get('secrets.pip-ss-kv.B2C_USERNAME'));
-          await signInPage.enterText(config.get('secrets.pip-ss-kv.B2C_USERNAME'), 'EmailField');
-          await signInPage.enterText(config.get('secrets.pip-ss-kv.B2C_PASSWORD'), 'PasswordField');
-          accountHomePage = await signInPage.clickSignIn();
-          await browser.pause(2000);
-        });
+      it('should select \'Sign in with my P&I details\' option, navigate to the login page, and sign in', async () => {
+        await signInPage.open('/sign-in');
+        await signInPage.selectOption('SignInRadio3');
+        await signInPage.clickContinueForRadio3();
+        console.log('B2C_USERNAME', process.env.B2C_USERNAME);
```

Comment:
console log left in

Reply:
this one is on purpose so we can see if there are any issues on the pipeline